src/components/views/read/PlayerView.js

```diff
@@ -85,17 +76,45 @@ const PlayerView = () => {
     }
   }, [inputResource]);
 
+  const adaptHeight = () => {
```

Comment:
Maybe we can optimize this so that it changes only when the height of the frame changes maybe using the ResizeObserver.